### PR TITLE
Added showcase pages map,table,chart

### PR DESCRIPTION
**Pull request by Rahul139-stack**

### CODE REVIEW

File: tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml [hunk 1]
```diff
@@ -1,17 +1,176 @@
...
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="typerefinery/components/layout/container"
+                    borderEnabled="false"
+                    classes="\0"
+                    flexEnabled="false"
+                    id="container_LANI4JGBE"
+                    marginEnabled="false"
+                    paddingEnabled="false"
+                    persistColorWhenThemeSwitches="false"
+                    title="Barchart Showcase">
```
Comment (Harikarthyk) [inline tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml:128]: This is for Line chart why is this Barchart here ?

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Please test before pushing.

Also take care of additional points.
1. Instead of using title component you can use section with title. Or If you are using title don't put container inside container.

File: tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml (hunk 1 above)
Comment (wildone) [reply tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml:128]: also title should just be Barchart, there is no reason to append " Showcase" as its given.

Comment (Rahul139-stack) [reply tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/widgets/chart/linechart/.content.xml]: Done, with single container having section with title

**Comment by Rahul139-stack** [comment]
> In the table showcase, I could not see any buttons on the action columns.
> 
> Can you please add an edit button to the table action columns ?
> 
> And when you click on that button, it should open a modal.

Done

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Why the action button tooltip is given random name ?

image

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**